projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -135,8 +319,12 @@ public function get_items( $request ) { // phpcs:ignore VariableAnalysis.CodeAna
 
 		return rest_ensure_response(
 			array(
-				'styles'  => $styles,
-				'scripts' => $scripts,
+				'allowed_block_types' => array_merge(
+					$this->get_core_block_types(),
+					self::ALLOWED_PLUGIN_BLOCKS
```

Comment:
Not really sure if this is strictly possible. Or if makes any difference to the mobile editor, 
but it would be good to check if a block is registered before returning in this list. 
There could be plugins that deregister blocks for example. So passing the 					self::ALLOWED_PLUGIN_BLOCKS though a `WP_Block_Type_Registry::get_instance()->is_registered( block )` filter would work here.